flytekit/core/context_manager.py

```diff
@@ -171,6 +174,7 @@ def __init__(self, current: typing.Optional[ExecutionParameters] = None):
             self.execution_id = current.execution_id if current else None
             self.logging = current.logging if current else None
             self.checkpoint = current._checkpoint if current else None
+            self.decks = current._decks if current else None
```

Comment:
```suggestion
            self.decks = current._decks if current else []
```

For type consistency